website/docs/r/lightsail_database.html.markdown

```diff
@@ -0,0 +1,115 @@
+---
+subcategory: "Lightsail"
+layout: "aws"
+page_title: "AWS: aws_lightsail_database"
+description: |-
+  Provides a Lightsail Database
+---
+
+# Resource: aws_lightsail_database
+
+Provides a Lightsail Database. Amazon Lightsail is a service to provide easy virtual private servers
+with custom software already setup. See [What is Amazon Lightsail?](https://lightsail.aws.amazon.com/ls/docs/getting-started/article/what-is-amazon-lightsail)
+for more information.
+
+~> **Note:** Lightsail is currently only supported in a limited number of AWS Regions, please see ["Regions and Availability Zones in Amazon Lightsail"](https://lightsail.aws.amazon.com/ls/docs/overview/article/understanding-regions-and-availability-zones-in-amazon-lightsail) for more details
+
+## Example Usage
+
+```terraform
```

Comment:
I would recommend one or more more complete examples with some of the attributes that might cause questions such as `preferred_backup_window` and `preferred_maintenance_window` or 1blueprint_id`.

Reply:
Added more complete examples to cover some of the advanced attributes.